bin/ttree

```diff
@@ -360,22 +362,34 @@ sub process_file {
     $dest = $destdir ? "$destdir/$destfile" : $destfile;
                    
 #    print "proc $file => $dest\n";
-    
-    # check against link list
-    foreach my $link_pattern (@$link) {
-        if ($filename =~ /$link_pattern/) {
-            $link_file = $copy_file = 1;
-            $check = $link_pattern;
-            last;
-        }
+
+    unless ($link_file) {
+	# check against link list
+	foreach my $link_pattern (@$link) {
+	    if ($filename =~ /$link_pattern/) {
+		$link_file = $copy_file = 1;
+		$check = "/$link_pattern/";
+		last;
+	    }
+	}
+    }
+
+    unless ($link_file) {
+	foreach my $asset_prefix (@$assets) {
+	    if ( index($file, "$asset_prefix/") == 0 ) {
```

Comment:
that final `/` makes an assumption on the operating system
I guess this is probably not the only issue in this codebase
using `File::Spec` avoid such issues and improve compatibility

Reply:
I'm not sure how File::Spec can help here... My usual approach is `=~ m;^\Q$asset_prefix\E[/\\];`.
Better still: leave it and add a line similar to line 56 of t/fileline.t...

Interesting note: File::Spec only uses slashes in the documentation and examples.

Reply:
Indeed `$asset_pefix` is an input from the user so it can contain any characters, and we can expect that the user will use what works for him.
Something like `File::Spec->catfile($asset_prefix, "")` would have added the file separator character for you.
But we can leave it like this and fix it when it becomes an issue... 
this is why I approved the request